.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,20 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
+'on':
+  push:
+    branches:
+      - master
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: npm run build
+      - uses: FirebaseExtended/action-hosting-deploy@v0
+        with:
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
Does this add the api keys?

Reply:
no this is basically only a token for the github permissions

Reply:
Where do we add it then? Loading the weather doesn't work yet on the deployed application

Reply:
They were added